### ISSUE TITLE
incompatible character encodings: UTF-8 and ASCII-8BIT

### ISSUE THREAD

**Issue by DunklerPhoenix** [joeyates/imap-backup#93]
Heho
I saw this error message in the log. Maybe it is a bug.

Complete output:
(Info: It should be "Gästebuch" not "G▒stebuch")
```
D, [2021-02-17T20:28:08.451049 #11456] DEBUG -- : [[Gmail]/Alle Nachrichten] uid: 68743 (1/314642) - 3720 bytes
W, [2021-02-17T20:28:08.453340 #11456]  WARN -- : [[Gmail]/Alle Nachrichten] failed to save message 68743:
Delivered-To: *REMOVED*
Received: by 10.202.189.86 with SMTP id n83csp579822oif;
Sat, 13 Jun 2015 20:59:47 -0700 (PDT)
X-Received: by 10.202.68.213 with SMTP id r204mr17546307oia.97.1434254286412;
Sat, 13 Jun 2015 20:58:06 -0700 (PDT)
Authentication-Results: mx.google.com;
spf=neutral (google.com: *REMOVED* is neither permitted nor denied by best guess record for domain of *REMOVED*+caf_=*REMOVED*=googlemail.com@*REMOVED*) smtp.mail=*REMOVED*+caf_=*REMOVED*=googlemail.com@*REMOVED*
Received-SPF: neutral (google.com: *REMOVED* is neither permitted nor denied by best guess record for domain of *REMOVED*+caf_=*REMOVED*=googlemail.com@*REMOVED*) client-ip=*REMOVED*;
Received: by 10.182.106.1 with POP3 id gq1mf3666619obb.3;
Sat, 13 Jun 2015 20:58:06 -0700 (PDT)
X-Gmail-Fetch-Info: *REMOVED*@googlemail.com 1 pop.googlemail.com 995 *REMOVED*
Delivered-To: *REMOVED*@gmail.com
Received: by 10.210.35.2 with SMTP id i2cs659777ebi;
Sun, 16 Aug 2009 07:15:58 -0700 (PDT)
Received: by 10.210.62.12 with SMTP id k12mr1600306eba.62.1250432158388;
Sun, 16 Aug 2009 07:15:58 -0700 (PDT)
Return-Path: <*REMOVED*+caf_=*REMOVED*=googlemail.com@*REMOVED*>
Received: from mail-ew0-f216.google.com (mail-ew0-f216.google.com [*REMOVED*])
by mx.google.com with ESMTP id 26si402441ewy.11.2009.08.16.07.15.58;
Sun, 16 Aug 2009 07:15:58 -0700 (PDT)
Received-SPF: neutral (google.com: *REMOVED* is neither permitted nor denied by best guess record for domain of *REMOVED*+caf_=*REMOVED*=googlemail.com@*REMOVED*) client-ip=*REMOVED*;
Authentication-Results: mx.google.com; spf=neutral (google.com: *REMOVED* is neither permitted nor denied by best guess record for domain of *REMOVED*+caf_=*REMOVED*=googlemail.com@*REMOVED*) smtp.mail=*REMOVED*+caf_=*REMOVED*=googlemail.com@*REMOVED*
Received: by mail-ew0-f216.google.com with SMTP id 12so2073190ewy.0
for <*REMOVED*@googlemail.com>; Sun, 16 Aug 2009 07:15:58 -0700 (PDT)
Received: by 10.210.62.4 with SMTP id k4mr5761384eba.25.1250432158284;
Sun, 16 Aug 2009 07:15:58 -0700 (PDT)
X-Forwarded-To: *REMOVED*@googlemail.com
X-Forwarded-For: *REMOVED*@*REMOVED* *REMOVED*@googlemail.com
Delivered-To: *REMOVED*@*REMOVED*
Received: by 10.211.165.2 with SMTP id s2cs944441ebo;
Sun, 16 Aug 2009 07:15:58 -0700 (PDT)
Received: by 10.204.65.1 with SMTP id g1mr350426bki.69.1250432157831;
Sun, 16 Aug 2009 07:15:57 -0700 (PDT)
Return-Path: <nobody@*REMOVED*>
Received: from *REMOVED* ([*REMOVED*])
by mx.google.com with ESMTP id 12si5087780bwz.2.2009.08.16.07.15.57;
Sun, 16 Aug 2009 07:15:57 -0700 (PDT)
Received-SPF: neutral (google.com: *REMOVED* is neither permitted nor denied by best guess record for domain of nobody@*REMOVED*) client-ip=*REMOVED*;
Received: by *REMOVED* (Postfix, from userid 65534)
id 3D9211383CC; Sun, 16 Aug 2009 16:15:57 +0200 (CEST)
To: *REMOVED*@*REMOVED*
Subject: =?iso-8859-1?Q?Neuer_G=E4stebuch-Eintrag?=
Date: Sun, 16 Aug 2009 16:15:57 +0200
From: =?iso-8859-1?Q?g=E4stebuch=40vollmond=2Eclan=2Ede?= <g▒stebuch@vollmond.clan.de>
Message-ID: <eda6a458639f7d1cb3f489f2e7ad9e6b@*REMOVED*>
X-Priority: 3
X-Mailer: PHPMailer [version 1.73]
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
Content-Type: text/plain; charset="iso-8859-1"
Sie haben einen neuen Eintrag in Ihrem G▒stebuch.
Name: 234242424243
Kommentar: fdgfdgdgd
Danke.
. incompatible character encodings: UTF-8 and ASCII-8BIT:
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/encodings.rb:137:in `join'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/encodings.rb:137:in `value_decode'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/encodings.rb:109:in `decode_encode'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/fields/unstructured_field.rb:84:in `do_decode'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/fields/unstructured_field.rb:62:in `decoded'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/fields/unstructured_field.rb:66:in `default'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/field.rb:239:in `method_missing'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/message.rb:1215:in `default'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/message.rb:666:in `from'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/email/mboxrd/message.rb:64:in `first_from'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/email/mboxrd/message.rb:55:in `best_from'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/email/mboxrd/message.rb:48:in `from'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/email/mboxrd/message.rb:26:in `to_serialized'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/serializer/mbox_store.rb:60:in `add'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/serializer/mbox.rb:43:in `save'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/downloader.rb:25:in `block in run'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/downloader.rb:15:in `each'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/downloader.rb:15:in `with_index'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/downloader.rb:15:in `run'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/account/connection.rb:67:in `block in run_backup'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/account/connection.rb:123:in `block in each_folder'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/account/connection.rb:120:in `each'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/account/connection.rb:120:in `each_folder'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/account/connection.rb:62:in `run_backup'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/configuration/list.rb:20:in `block in each_connection'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/configuration/list.rb:18:in `each'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/lib/imap/backup/configuration/list.rb:18:in `each_connection'
/var/lib/gems/2.5.0/gems/imap-backup-3.2.0/bin/imap-backup:76:in `<top (required)>'
/usr/local/bin/imap-backup:23:in `load'
/usr/local/bin/imap-backup:23:in `<main>'"

```

**Comment by joeyates** [comment]
Thanks @DunklerPhoenix 

I'll work on a fix for that.

**Comment by joeyates** [comment]
@DunklerPhoenix imap-backup >= 4 with debugging enabled now outputs shows the full calls and responses between the library and the IMAP server. If you are able to use that to capture the exact content of emails that fail with this error, that would be very useful.

**Comment by DunklerPhoenix** [comment]
I thought I answered :D I'll wrote: "I'll take a look. Just have much to do atm".


Anyway. Today I looked over it and the broken email still showing up:
```

X-Priority: 3
X-Mailer: PHPMailer [version 1.73]
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
Content-Type: text/plain; charset="iso-8859-1"
Sie haben einen neuen Eintrag in Ihrem G䳴ebuch.
Name: 234242424243
Kommentar: fdgfdgdgd
Danke.
. incompatible character encodings: UTF-8 and ASCII-8BIT:
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/encodings.rb:137:in `join'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/encodings.rb:137:in `value_decode'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/encodings.rb:109:in `decode_encode'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/fields/unstructured_field.rb:84:in `do_decode'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/fields/unstructured_field.rb:62:in `decoded'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/fields/unstructured_field.rb:66:in `default'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/field.rb:239:in `method_missing'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/message.rb:1215:in `default'
/var/lib/gems/2.5.0/gems/mail-2.7.1/lib/mail/message.rb:666:in `from'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/email/mboxrd/message.rb:68:in `first_from'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/email/mboxrd/message.rb:59:in `best_from'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/email/mboxrd/message.rb:52:in `from'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/email/mboxrd/message.rb:30:in `to_serialized'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/serializer/mbox_store.rb:59:in `add'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/serializer/mbox.rb:48:in `save'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/downloader.rb:25:in `block in run'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/downloader.rb:15:in `each'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/downloader.rb:15:in `with_index'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/downloader.rb:15:in `run'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/account/connection.rb:63:in `block in run_backup'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/account/connection.rb:129:in `block in each_folder'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/account/connection.rb:126:in `each'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/account/connection.rb:126:in `each_folder'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/account/connection.rb:57:in `run_backup'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/cli/backup.rb:16:in `block in run'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/cli/helpers.rb:30:in `block in each_connection'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/configuration/list.rb:21:in `block in each_connection'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/configuration/list.rb:19:in `each'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/configuration/list.rb:19:in `each_connection'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/cli/helpers.rb:29:in `each_connection'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/cli/backup.rb:15:in `run'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/lib/imap/backup/cli.rb:45:in `backup'
/var/lib/gems/2.5.0/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
/var/lib/gems/2.5.0/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
/var/lib/gems/2.5.0/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
/var/lib/gems/2.5.0/gems/thor-1.1.0/lib/thor/base.rb:485:in `start'
/var/lib/gems/2.5.0/gems/imap-backup-4.0.3/bin/imap-backup:8:in `<top (required)>'
/usr/local/bin/imap-backup:23:in `load'
/usr/local/bin/imap-backup:23:in `<main>'"

```

**Comment by joeyates** [comment]
I have been unable to reproduce this issue. I suspect it happens only with certain IMAP server implementations and with certain character encodings. Closing for now.

**Comment by hakamairi** [comment]
Hi @joeyates, I got the same error with current version while backuping gmail account. Could there be a flag/setting to ignore such emails? 
The only workaround I found is to delete such email from source server.

Kind regards,

**Comment by bderubinat** [comment]
Hi @joeyates 

I'm having the same issue.

a) Could you add any logging for us to figure out which email it is ? So we can provide info (or the email) to reproduce.

b) could you not stop the backup process but rather continue and give the number of errors at the end ?


`
E, [2023-10-09T02:12:27.981005 #55162] ERROR -- : Backup for account '<snip>' failed with error incompatible character encodings: UTF-8 and ASCII-8BIT
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/mail-2.7.1/lib/mail/encodings.rb:137:in `join'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/mail-2.7.1/lib/mail/encodings.rb:137:in `value_decode'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/mail-2.7.1/lib/mail/encodings.rb:109:in `decode_encode'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/mail-2.7.1/lib/mail/fields/unstructured_field.rb:84:in `do_decode'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/mail-2.7.1/lib/mail/fields/unstructured_field.rb:62:in `decoded'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/mail-2.7.1/lib/mail/fields/unstructured_field.rb:66:in `default'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/mail-2.7.1/lib/mail/field.rb:239:in `method_missing'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/mail-2.7.1/lib/mail/message.rb:1215:in `default'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/mail-2.7.1/lib/mail/message.rb:666:in `from'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/email/mboxrd/message.rb:75:in `first_from'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/email/mboxrd/message.rb:66:in `best_from'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/email/mboxrd/message.rb:59:in `from'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/email/mboxrd/message.rb:31:in `to_serialized'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/serializer/delayed_metadata_serializer.rb:35:in `append'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:95:in `handle_uid_and_body'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:78:in `block in download_block'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:77:in `each'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:77:in `with_index'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:77:in `download_block'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:32:in `block in run'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:31:in `each'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:31:in `each_slice'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:31:in `with_index'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/downloader.rb:31:in `run'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/account/folder_backup.rb:31:in `block in run'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/serializer/delayed_metadata_serializer.rb:25:in `block (2 levels) in transaction'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/serializer/mbox.rb:19:in `block in transaction'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/serializer/transaction.rb:17:in `begin'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/serializer/mbox.rb:18:in `transaction'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/serializer/delayed_metadata_serializer.rb:24:in `block in transaction'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/serializer/transaction.rb:17:in `begin'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/serializer/delayed_metadata_serializer.rb:23:in `transaction'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/account/folder_backup.rb:30:in `run'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/account/backup.rb:35:in `block in run'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/account/backup_folders.rb:39:in `block in each'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/account/backup_folders.rb:39:in `each'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/account/backup_folders.rb:39:in `each'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/account/backup.rb:34:in `run'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/cli/backup.rb:29:in `block in run'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/cli/backup.rb:27:in `each'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/cli/backup.rb:27:in `run'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/cli.rb:92:in `backup'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/thor-1.2.2/lib/thor/command.rb:27:in `run'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/thor-1.2.2/lib/thor/invocation.rb:127:in `invoke_command'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/thor-1.2.2/lib/thor.rb:392:in `dispatch'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/thor-1.2.2/lib/thor/base.rb:485:in `start'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/cli.rb:65:in `start'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/bin/imap-backup:12:in `block in <top (required)>'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/lib/imap/backup/logger.rb:41:in `sanitize_stderr'
/usr/local/Cellar/imap-backup/13.0.0/libexec/gems/imap-backup-13.0.0/bin/imap-backup:11:in `<top (required)>'
/usr/local/Cellar/imap-backup/13.0.0/libexec/bin/imap-backup:25:in `load'
/usr/local/Cellar/imap-backup/13.0.0/libexec/bin/imap-backup:25:in `<main>'
`

**Comment by joeyates** [comment]
Hi @bderubinat 

To answer your first question, you can run the backup with `-vv` to see internal logging and all communications with the IMAP server.

Skipping problematic emails is next on the ROADMAP.

**Comment by bderubinat** [comment]
Thanks @joeyates 

Good for the roadmap ;-)

As for the problematic email, I now know it is in the Junk folder.. Excluding the folder makes it all good.

An badly formated is probably not that much of an issue (besides halting the backup).

Maybe "Junk" should be excluded folder by default ;-)

extract of the problematic email anyway
```
S: From: "=?windows-1252?Q?Curso_2_en_1:_Gr=E1matica_y_Ortograf=EDa_y_Redacci=F3n?=" <redacci�nF6@ilearnexperience.com>
S: To: < <snip> >
S: Subject: =?windows-1252?Q?Gram=E1tica_funcional,_enunciados_y_p=E1rrafos?=
S: Date: Mon, 6 Feb 2023 12:21:06 -0600
S: X-Priority: 1
S: X-Mailer: SMTP COMPONENT
S: MIME-Version: 1.0
S: Content-Type: multipart/alternative;
S: 	boundary="----=SPLITOR00A_001_481655890D"
S: X-GND-Status: MCE
S: Received-SPF: pass (spool3: domain of ilearnexperience.com designates 92.204.175.108 as permitted sender) client-ip=92.204.175.108; envelope-from=redacci�nF6@ilearnexperience.com; helo=mail2.ilearnexperience.com;
S: Authentication-Results: spool.mail.gandi.net;
S: 	dkim=none;
S: 	dmarc=none;
S: 	spf=none (spool.mail.gandi.net: domain of mail2.ilearnexperience.com has no SPF policy when checking 92.204.175.108) smtp.helo=mail2.ilearnexperience.com
S:
S: This is a multi-part message in MIME format.
S:
S: ------=SPLITOR00A_001_481655890D
S: Content-Type: text/plain;
S: 	charset="windows-1252"
S: Content-Transfer-Encoding: base64
S:
S: DQpFc3RlIGN1cnNvIGF5dWRhcuEgYSBsb3MgcGFydGljaXBhbnRlcyBhIGVudGVuZGVyIGxhcyBi
S: YXNlcyBwYXJhIHJlYWxpemFyIGVzY3JpdG9zIGVmaWNhY2VzIGVuIGVsIOFtYml0byBsYWJvcmFs
S: LiANCiANCkN1cnNvIDIgZW4gMTogVGFsbGVyIGRlIGdyYW3hdGljYSB5IG9ydG9ncmFm7WEgcGFy
S: YSBwcm9mZXNpb25pc3RhcyB5IFRhbGxlciBkZSByZWRhY2Np824gcGFyYSBwcm9mZXNpb25pc3Rh
S: cw0KDQpXZWJpbmFyIGVuIHZpdm8NCkluaWNpbzoxMSBkZSBGZWJyZXJvIC0gRmluOiAxMSBkZSBN
S: YXJ6bw0KU+FiYWRvcyBkZSAxMDowMCBhIDE0OjAwIFBNLiANCk90b3JnYXJlbW9zIGEgbG9zIHBh
S: cnRpY2lwYW50ZXMgaW5mb3JtYWNp824gdmVyYXogeSBhY3R1YWxpemFkYSwgaGVycmFtaWVudGFz
S: IHkgY29ub2NpbWllbnRvcyBi4XNpY29zIG5lY2VzYXJpb3MgcGFyYSB1bmEgY29ycmVjdGEgcmVk
S: YWNjafNuIGRlIHRleHRvcyBwcm9mZXNpb25hbGVzIGUgaW5mb3JtYWxlcyBxdWUgbGVzIHBlcm1p
S: dGFuIHVuYSBjb211bmljYWNp824gcG9yIGVzY3JpdG8gY2xhcmEsIGNvcnJlY3RhIHkgZWZpY2F6
S: LiANCiANCiANClJhem9uZXMgcGFyYSBwYXJ0aWNpcGFyOg0KDQotIENvbm9jZXLhIHkgZW50ZW5k
S: ZXLhIGxhcyBub2Npb25lcyBt4XMgaW1wb3J0YW50ZXMgZGUgbGEgZ3JhbeF0aWNhIGRlbCBpZGlv
S: bWEgZXNwYfFvbC4NCi0gRW5saXN0YXLhIHkgcmV2aXNhcuEgbGFzIGNhcmFjdGVy7XN0aWNhcyB5
S: IHByb3BpZWRhZGVzIGRlIGxhcyBsZXRyYXMsIHBhbGFicmFzIHkgb3JhY2lvbmVzLg0KLSBDb25v
S: Y2Vy4SBsb3MgYXNwZWN0b3MgdGXzcmljb3MgaW5kaXNwZW5zYWJsZXMgcGFyYSB1bmEgY29tcHJl
S: bnNp824gaW50ZWdyYWwgZGVsIG9yaWdlbiBkZSBsYXMgcmVnbGFzIG9ydG9ncuFmaWNhcy4NCg0K
S: DQpZIG11Y2hvIG3hcy4NCiANClNvbGljaXRhciBUZW1hcmlvIENvbXBsZXRvIA0KDQogDQpQYXJh
S: IG1heW9yIGluZm9ybWFjafNuLCByZXNwb25kZXIgc29icmUgZXN0ZSBjb3JyZW8NCmNvbiBsYSBw
S: YWxhYnJhOkdSZWRhY2Np824gKyBsb3Mgc2lndWllbnRlcyBkYXRvczogDQogDQpOb21icmU6ICBU
S: ZWzpZm9ubyAvIFdoYXRzQXBwOiAgRW1wcmVzYTogIA0KDQpQYXJhIGluZm9ybWFjafNuIGlubWVk
S: aWF0YSBsbGFtYXIgYWw6DQooKzUyKSA1NSAxNSA1NCA2NiAzMCAtICgrNTIpIDU1IDMwIDE2IDU5
S: IDQzDQpPIHB1ZWRlIGVudmlhcm5vcyB1biBtZW5zYWplIHbtYSBXaGF0c0FwcCANCiANCiANCg0K
S: DQoNCi0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0t
S: LS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tDQpJbm5vdmEgTGVhcm4gTel4aWNvIC0gaW5ub3Zh
S: bGVhcm4uIG14IC0gTelyaWRhLCBZdWNhdOFuLCBN6XhpY28uDQpTaSBkZXNlYSBkZWphciBkZSBy
S: ZWNpYmlyIG51ZXN0cmEgcHJvbW9jafNuIGZhdm9yIGRlIHJlc3BvbmRlciBjb24gbGEgcGFsYWJy
S: YSBiYWphIG8gZW52aWFyIHVuIGNvcnJlbyBhIGJhamFzQCBpbm5vdmFsZWFybi5uZXQgIA0KIA0K
S:
S: ------=SPLITOR00A_001_481655890D
S: Content-Type: text/html;
S: 	charset="windows-1252"
S: Content-Transfer-Encoding: base64
S:
S: PGh0bWwgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGh0bWwiPjxoZWFkPjx0aXRsZT5n
S: cmFtYXRpY2E8L3RpdGxlPg0KPG1ldGEgY29udGVudD0idGV4dC9odG1sOyBjaGFyc2V0PXV0Zi04

```

**Comment by joeyates** [comment]
@bderubinat I've released version 14.1.0 which skips emails that cause serialization errors.
